### DISCUSSION TITLE
Where I can set the workspace/configuration request's response?

### DISCUSSION THREAD

**Discussion by nluyan** [TypeFox/monaco-languageclient#619] [General]
I tried to connect monaco-languageclient to a csharp language server(modified from the python sample), but when the language server send a workspace/configuration request,  like this:
`{"jsonrpc":"2.0","id":3,"method":"workspace/configuration","params":{"items":[{"section":"csharp|symbol_search.dotnet_search_reference_assemblies"},{"section":"visual_basic|symbol_search.dotnet_search_reference_assemblies"},{"section":"csharp|implement_type.dotnet_insertion_behavior"},{"section":"visual_basic|implement_type.dotnet_insertion_behavior"},{"section":"csharp|implement_type.dotnet_property_generation_behavior"},{"section":"visual_basic|implement_type.dotnet_property_generation_behavior"},{"section":"csharp|completion.dotnet_show_name_completion_suggestions"},{"section":"visual_basic|completion.dotnet_show_name_completion_suggestions"},{"section":"csharp|completion.dotnet_provide_regex_completions"},{"section":"visual_basic|completion.dotnet_provide_regex_completions"},{"section":"csharp|completion.dotnet_show_completion_items_from_unimported_namespaces"},{"section":"visual_basic|completion.dotnet_show_completion_items_from_unimported_namespaces"},{"section":"csharp|quick_info.dotnet_show_remarks_in_quick_info"},{"section":"visual_basic|quick_info.dotnet_show_remarks_in_quick_info"},{"section":"navigation.dotnet_navigate_to_decompiled_sources"},{"section":"csharp|highlighting.dotnet_highlight_related_json_components"},{"section":"visual_basic|highlighting.dotnet_highlight_related_json_components"},{"section":"csharp|highlighting.dotnet_highlight_related_regex_components"},{"section":"visual_basic|highlighting.dotnet_highlight_related_regex_components"},{"section":"csharp|inlay_hints.dotnet_enable_inlay_hints_for_parameters"},{"section":"visual_basic|inlay_hints.dotnet_enable_inlay_hints_for_parameters"},{"section":"csharp|inlay_hints.dotnet_enable_inlay_hints_for_literal_parameters"},{"section":"visual_basic|inlay_hints.dotnet_enable_inlay_hints_for_literal_parameters"},{"section":"csharp|inlay_hints.dotnet_enable_inlay_hints_for_indexer_parameters"},{"section":"visual_basic|inlay_hints.dotnet_enable_inlay_hints_for_indexer_parameters"},{"section":"csharp|inlay_hints.dotnet_enable_inlay_hints_for_object_creation_parameters"},{"section":"visual_basic|inlay_hints.dotnet_enable_inlay_hints_for_object_creation_parameters"},{"section":"csharp|inlay_hints.dotnet_enable_inlay_hints_for_other_parameters"},{"section":"visual_basic|inlay_hints.dotnet_enable_inlay_hints_for_other_parameters"},{"section":"csharp|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_differ_only_by_suffix"},{"section":"visual_basic|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_differ_only_by_suffix"},{"section":"csharp|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_match_method_intent"},{"section":"visual_basic|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_match_method_intent"},{"section":"csharp|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_match_argument_name"},{"section":"visual_basic|inlay_hints.dotnet_suppress_inlay_hints_for_parameters_that_match_argument_name"},{"section":"csharp|inlay_hints.csharp_enable_inlay_hints_for_types"},{"section":"visual_basic|inlay_hints.csharp_enable_inlay_hints_for_types"},{"section":"csharp|inlay_hints.csharp_enable_inlay_hints_for_implicit_variable_types"},{"section":"visual_basic|inlay_hints.csharp_enable_inlay_hints_for_implicit_variable_types"},{"section":"csharp|inlay_hints.csharp_enable_inlay_hints_for_lambda_parameter_types"},{"section":"visual_basic|inlay_hints.csharp_enable_inlay_hints_for_lambda_parameter_types"},{"section":"csharp|inlay_hints.csharp_enable_inlay_hints_for_implicit_object_creation"},{"section":"visual_basic|inlay_hints.csharp_enable_inlay_hints_for_implicit_object_creation"},{"section":"csharp|code_style.formatting.indentation_and_spacing.tab_width"},{"section":"visual_basic|code_style.formatting.indentation_and_spacing.tab_width"},{"section":"csharp|code_style.formatting.indentation_and_spacing.indent_size"},{"section":"visual_basic|code_style.formatting.indentation_and_spacing.indent_size"},{"section":"csharp|code_style.formatting.indentation_and_spacing.indent_style"},{"section":"visual_basic|code_style.formatting.indentation_and_spacing.indent_style"},{"section":"csharp|code_style.formatting.new_line.end_of_line"},{"section":"visual_basic|code_style.formatting.new_line.end_of_line"},{"section":"code_style.formatting.new_line.insert_final_newline"},{"section":"csharp|background_analysis.dotnet_analyzer_diagnostics_scope"},{"section":"visual_basic|background_analysis.dotnet_analyzer_diagnostics_scope"},{"section":"csharp|background_analysis.dotnet_compiler_diagnostics_scope"},{"section":"visual_basic|background_analysis.dotnet_compiler_diagnostics_scope"},{"section":"csharp|code_lens.dotnet_enable_references_code_lens"},{"section":"visual_basic|code_lens.dotnet_enable_references_code_lens"},{"section":"csharp|code_lens.dotnet_enable_tests_code_lens"},{"section":"visual_basic|code_lens.dotnet_enable_tests_code_lens"},{"section":"projects.dotnet_binary_log_path"},{"section":"projects.dotnet_enable_automatic_restore"}]}}`

then the client responsed a message like this:
`
 {"jsonrpc":"2.0","id":3,"result":[null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null,null]}
`
but it suppose to response a message like this:
`
[true,null,"withOtherMembersOfTheSameKind",null,"preferThrowingProperties",null,"true",null,"true",null,true,null,"true",null,"true","true",null,"true",null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,false,null,"4",null,"4",null,"space",null,"auto",null,false,"openFiles",null,"openFiles",null,true,null,true,null,null,true]}
`
How can I do to set the client's workspace/configuration param value?
Thank you!

**Comment by kaisalmen** [comment]
@nluyan see python client config being mentioned here: https://github.com/TypeFox/monaco-languageclient/discussions/616#discussioncomment-8874628
